resources/views/timelines/_timeline.blade.php

```diff
@@ -94,11 +94,32 @@ class="btn btn-box-tool" role="button"
     @endcan
     </ul>
 @empty
+    @php
+    $position = 0;
+    @endphp
     <div class="alert alert-warning">
-        <p>{{ __('timelines.helpers.no_era') }}</p>
+        <p>{{ __('timelines.helpers.no_era') }} </p>
+        @can('update', $timeline)
+        <br>  
+        <a href="{{ route('timelines.timeline_eras.create', ['timeline' => $model, 'from' => 'view']) }}" class="btn btn-warning btn-sm"
+            data-url="{{ route('timelines.timeline_eras.create', ['timeline' => $model]) }}"
+            data-backdrop="static">
+            <i class="fa-solid fa-plus"></i> {{ __('timelines/eras.actions.add') }}
+        </a>
+        @endcan
     </div>
 @endforelse
-
+@if($position)
```

Comment:
We can skip using a new variable by simply checking `!$timeline->eras->isEmpty()`. Also, don't forget space between conditions and the first ( :)